app/usecases/clans.py

```diff
@@ -1,17 +1,32 @@
 from __future__ import annotations
 
-import asyncio
+from datetime import timedelta
+from typing import Optional
 
 import app.state
-import logger
 
 CLANS: dict[int, str] = {}
 FIVE_MINUTES = 60 * 5
 
 
+async def cache(user_id: int, clan_tag: str) -> None:
+    await app.state.services.redis.set(
```

Comment:
you should use kwargs here cus it looks ugly reading flow without being told what each arg is for

Reply:
Fixed